charts/argo-cd/values.yaml

```diff
@@ -653,6 +653,8 @@ server:
   #      oauthclientCredentials:
   #        secretName: argocd-secret
 
+  extraContainers: []
```

Comment:
Could we get an example of an extra container in the comments below this entry here so the purpose is clear.

Reply:
Added :-)